app/javascript/components/download/Download.vue

```diff
@@ -41,7 +41,12 @@ export default {
     textCommercial: {
       required: true,
       type: Object //See download_text in downloads_helper.rb
-    }
+    },
+    downloadDisabled: {
+      default: false,
+      required: false,
```

Comment:
I think the `Download` component is being used in a number of places across the website, so I felt the `required: false` line was important